### DISCUSSION TITLE
It's possible to use NextJS with Puppeteer?

### DISCUSSION THREAD

**Discussion by omariosouto** [vercel/next.js#17537] [Help]
I'm trying to do some web scrapping with puppeteer and NextJS, but I'm not having success deploying it. Does anyone have success in it?

**Comment by bajramdev** [comment]
did you solve this ?

> I'm trying to do some web scrapping with puppeteer and NextJS, but I'm not having success deploying it. Does anyone have success in it?

Checkout MonoRepos from Vercel https://vercel.com/blog/monorepos
